gym/wrappers/normalize.py

```diff
@@ -137,25 +127,16 @@ def __init__(
 
     def step(self, action):
         """Steps through the environment, normalizing the rewards returned."""
-        obs, rews, terminateds, truncateds, infos = step_api_compatibility(
-            self.env.step(action), True, self.is_vector_env
-        )
+        obs, rews, terminateds, truncateds, infos = self.env.step(action)
         if not self.is_vector_env:
             rews = np.array([rews])
         self.returns = self.returns * self.gamma + rews
         rews = self.normalize(rews)
-        if not self.is_vector_env:
-            dones = terminateds or truncateds
-        else:
-            dones = np.bitwise_or(terminateds, truncateds)
+        dones = np.logical_or(terminateds, truncateds)
```

Comment:
Why this change? It used to be either `or` or `np.bitwise_or`, now it's `np.logical_or`

Reply:
It simplifies the code. I checked, it works for both `True` and `np.array([True, False])`